cdap-app-fabric/src/main/java/io/cdap/cdap/internal/app/store/AppMetadataStore.java

```diff
@@ -431,6 +456,15 @@ public ApplicationMeta getLatest(ApplicationReference appReference) throws IOExc
     return null;
   }
 
+  /**
```

Comment:
Thanks for filling out these missing javadocs, this is definitely valuable to fix. Though in the future it would be good to do these types of improvements in a separate PR to keep things focused. For example, if we need to revert the commit for whatever reason, the javadoc improvements wouldn't need to be reverted along with it.

Reply:
Ah right. Will keep it in mind. Thanks